synapse/storage/database.py

```diff
@@ -2395,6 +2395,7 @@ def get_cache_dict(
             min_val = min(cache.values()) + 1
         else:
             min_val = max_value
+        logger.debug("Calling get_cache_dict %s: %s", table, sql)
```

Comment:
I think we want this logging to be similar to [`_load_current_id`](https://github.com/matrix-org/synapse/blob/02f74f3a997a4356b5bda957ebc51a829dad15f9/synapse/storage/util/id_generators.py#L78), so maybe something like:

```suggestion
        logger.debug("Initialising cached dict for %s(%s): %i", table, stream_column, min_val)
```